### ISSUE TITLE
Frost Nova visual snare effect doesn't fade when target is polymorphed

### ISSUE THREAD

**Issue by ifyousayso** [LightsHope/server#1226]
.. which is _weird_ considering it works perfectly for targets snared by Frostbite (talent).

**Comment by Cabro** [comment]
This affects all spells: Frost Nova, Hurricane, Tranquility, Roots (the sparks remain) etc. It's a new bug with the recent patch changes, and could easily be found or fixed.

**Comment by Cabro** [comment]
I don't think so, becuase this bug wasnt presnet until a few weeks ago. Definitely something that was changed in the recent patches.

**Comment by stesve** [comment]
Or automated (graphic driver) updates broke the client. Since only some people are experiencing it, and there have been quite a few reports about reverting to an older driver fixes it.

**Comment by Cabro** [comment]
That's it out of the option, there are no automatic video driver updates here. The version i use is from 2016. Like i said, it's something from the recent patches.

**Comment by stesve** [comment]
That only affects some clients. Nahhh....

Btw.. ms pushed an update late last year that made changes to the dx9 API.

I dont know if that update is causing it, but it tells me one thing; that ppl are still struggling to get old xp games running perfectly on a modern system.

**Comment by ifyousayso** [comment]
I have no updates on anything.. killed Windows Update too as soon as I install the "OS".

**Comment by Cabro** [comment]
Could it just be that the server is not sending to the clients the package info that tells the animation should be removed? Maybe?

**Comment by ifyousayso** [comment]
Also, I just relogged and the effect was still there.. seems highly unlikely that the client cached that..
